blockx/assetlist.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "blockx-atlantis-testnet",
+  "assets": [
+    {
+      "description": "BlockX Native Token",
+      "denom_units": [
+        {
+          "denom": "abcx",
+          "exponent": 0
+        },
+        {
+          "denom": "bcx",
+          "exponent": 18
+        }
+      ],
+      "base": "abcx",
+      "name": "BCX",
+      "display": "BCX",
```

Comment:
denoms are case sensitive and there is no denom matching this exactly. I think this `display` value should be changed to lowercase since that is defined

Reply:
Thanks so much, appreciate your time reviewing!
Excuse me for missing this.
Resolved with thanks.